### PR TITLE
adds in linter for engineers, design and copy resources

### PR DESCRIPTION
**Pull request by cheapwebmonkey**
This adds links and resources for in-editor linting, plus Design and copywriting resources for accessibility. There is also a card in the backlog to make Trello cards for these items.

### CODE REVIEW

File: _guide/onboarding-staff.md [hunk 1]
```diff
@@ -47,9 +47,26 @@ Our team has been working to extend the [CivicActions Accessibility](https://acc
...
+Install an accessibility browser extension. Take the time to learn the basics of whichever one you choose because everyone has a role in producing accessible websites.  
+  * [axe DevTool's Web Accessibility Testing for Chrome](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd)  
   * [WAVE](https://www.youtube.com/watch?v=ITUDiTgAZY0&t=270s)
-  * [Accessibility Insights](https://www.youtube.com/watch?v=U6NY8Cxym5g)
+  * [Accessibility Insights](https://www.youtube.com/watch?v=U6NY8Cxym5g)  
+  
+
+### Trello card - Install a color contrast checker  
+Install a color contrast checker browser extension, such as [ColorZilla for Chrome](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp), [ColorZilla for Chrome](https://addons.mozilla.org/en-US/firefox/addon/colorzilla/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search) or use a free site like [Coolors Color Contrast Checker](https://coolors.co/contrast-checker/112a46-acc8e5)
```
[the remark this answers is not in the document]
Comment (cheapwebmonkey) [reply _guide/onboarding-staff.md]: @dmundra is it possible to run the remark-linter locally?

[the remark this answers is not in the document]
Comment (dmundra) [reply _guide/onboarding-staff.md]: I believe it is via docker. I haven't done it so need to produce some steps but the linter script is located in the `lint` folder.

[the remark this answers is not in the document]
Comment (dmundra) [reply _guide/onboarding-staff.md]: Here are the steps after checking out the repo and branch

```
docker build -t accessibility-linter lint/. # Build the linter docker image
docker run -v $(pwd):/src/. --rm accessibility-linter lint/remark.sh # Run docker mounting the current directory and using the image built in the previous command
```

**Comment by mgifford** [comment]
@cheapwebmonkey My only thought so far is that not everyone uses Figma or a text editor for their work. Would be good to just add (if you use ... then) so that developers aren't trying to figure out Figma & designers aren't trying to figure out what a linter is.

File: .github/workflows/qa.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ jobs:
           time: '2m'
 
       - name: Comment on pull request.
-        uses: thollander/actions-comment-pull-request@master
+        uses: thollander/actions-comment-pull-request@v1
```
[the remark this answers is not in the document]
Comment (mgifford) [reply .github/workflows/qa.yml:58]: Sneaky.

**Comment by mgifford** [comment]
@cheapwebmonkey & @dmundra would be good to give this a quick review to see the modifications I proposed make sense & then let's push this into the site. At this point it has been open for too long. We can fix it later if others have concerns. These are great changes. Thanks!

**Review by mgifford — COMMENTED** [review]
Made some changes. Looks good.  Sorry I missed that this was waiting for my review.

**Review by dmundra — APPROVED** [review]
Looks good to me.

File: _guide/onboarding-staff.md [hunk 3]
```diff
@@ -44,73 +44,86 @@ Curious how some other people engage with the web?
...
 
 ### Trello card - Investigate and possibly configure accessibility settings in your operating system
 
 Apple, Windows, Chromebook, Linux desktop OS all have built-in accessibility features. So do mobile systems like Android & iOS.
 
 **Mac users** - enable [Full Keyboard Access](https://iainbean.com/posts/2020/an-opinionated-guide-to-accessibility-testing/#1.-The-Tab-key) to be able to test keyboard only navigation.
 
-
-### Trello card - Take a simple quiz with multiple-choice questions (include links to Google Forms):
+### Trello card - Take a this quiz with multiple-choice questions (include links to Google Forms):
```
Comment (jaube05) [inline _guide/onboarding-staff.md:77]: Small typo here, need to remove 'a' after 'Take'

File: _guide/onboarding-staff.md [hunk 4]
```diff
@@ -44,73 +44,86 @@ Curious how some other people engage with the web?
...
+* [WebAim's WAVE](https://wave.webaim.org/) - [Youtube: Intro to WAVE](https://www.youtube.com/watch?v=ITUDiTgAZY0&t=270s)
+* [Microsoft's Accessibility Insights](https://accessibilityinsights.io/) - [Youtube: Intro to Accessibility Insights](https://www.youtube.com/watch?v=U6NY8Cxym5g)
+
+### Trello card - Install a color contrast checker
+
+Install a color contrast checker browser extension, such as [ColorZilla for Chrome](https://chrome.google.com/webstore/detail/colorzilla/bhlhnicpbhignbdhedgjhgdocnmhomnp), [ColorZilla for Firefox](https://addons.mozilla.org/en-US/firefox/addon/colorzilla/) or use a free site like [Coolors Color Contrast Checker](https://coolors.co/contrast-checker/112a46-acc8e5)
+
+### Trello card - Developers, install an accessibility linter in your text editor or Integrated Development Environment (IDE)
+
+Install an accessibility linter in your text editor, such as [axe Accessibility Linter for VS Code - recommended](https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter). Please note, an accessiblity linter is required for front end dev work.
```
Comment (jaube05) [inline _guide/onboarding-staff.md:61]: Do we have one for phpstorm or a link for other IDE's developers may use

File: _guide/onboarding-staff.md [hunk 5]
```diff
@@ -17,23 +16,24 @@ our focus is on web accessibility. Making our content accessible is a key part o
...
 chance someone on your team will have a disability.
 
-
 ## Pre-workshop Tasks
 
 ### Trello card - Watch these short videos from this list:
 
 * [Deque Systems - What is Digital Accessibility?](https://www.youtube.com/watch?v=grrx2Lva7T0) (<3 minutes)
-* [W3C WAI, Shadi Abou-Zahra - Introduction to Web Accessibility and W3C Standards](https://www.youtube.com/watch?v=20SHvU2PKsM)  (<5 minutes)
+* [W3C WAI, Shadi Abou-Zahra - Introduction to Web Accessibility and W3C Standards](https://www.youtube.com/watch?v=20SHvU2PKsM) (<5 minutes)
```
[the remark this answers is not in the document]
Comment (mgifford) [reply _guide/onboarding-staff.md:24]: Might depend on the video. Can you find it @jaube05 ? We can always add it in later.  In this case, there's just an extra space that is being nixed, so we don't want to hold up the patch.

[the remark this answers is not in the document]
Comment (jaube05) [reply _guide/onboarding-staff.md:24]: We dont need to hold up the patch but here is the video https://www.youtube.com/watch?v=Gv1aDEFlXq8&t=4s

[the remark this answers is not in the document]
Comment (mgifford) [reply _guide/onboarding-staff.md:24]: I was confused by Nunavut @jaube05 — I now totally understand the confusion now. It was https://www.intuit.com employees. 

Yes. That should be in the onboarding. @cheapwebmonkey do you want to add it here or in another patch?

File: _guide/onboarding-staff.md (hunk 4 above)
Comment (cheapwebmonkey) [reply _guide/onboarding-staff.md:61]: There is nothing like this that I have been able to find for PHP Storm

File: _guide/onboarding-staff.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (cheapwebmonkey) [reply _guide/onboarding-staff.md:24]: Another patch please

**Review by mgifford — APPROVED** [review]
Looks good to me. @cheapwebmonkey please merge this issue if you're comfortable with it all.

**Pull request merged**